extensions/hibernate-search-standalone-elasticsearch/runtime/src/main/java/io/quarkus/hibernate/search/standalone/elasticsearch/SearchExtension.java

```diff
@@ -0,0 +1,69 @@
+package io.quarkus.hibernate.search.standalone.elasticsearch;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Repeatable;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import jakarta.enterprise.util.AnnotationLiteral;
+import jakarta.inject.Qualifier;
+
+/**
+ * CDI qualifier for beans representing an "extension" of Hibernate Search Standalone,
+ * i.e. beans injected into Hibernate Search as part of its configuration.
+ * <p>
+ * See the reference documentation for information about extensions that supports this annotation.
+ */
+@Target({ TYPE, FIELD, METHOD, PARAMETER })
+@Retention(RUNTIME)
+@Documented
+@Qualifier
+@Repeatable(SearchExtension.List.class)
+public @interface SearchExtension {
```

Comment:
You think it's not going to be confusing to have them with the exact same name and just different package?

No strong opinion on my side, just asking.

Reply:
It may be... but:

1. I don't expect usage of both extensions in the same app to be very frequent.
2. If people do that, they'll have naming conflicts all over the place already -- be it simply because of `SearchMapping` and `SearchSession`, defined in different packages in each extension.
3. If we were to use different names, I'd rename the ORM one too: `@SearchOrmExtension` vs. `@SearchStandaloneExtension`.

So I think if we want a solution for this problem, we'll have to think more broadly about it... later.